protocol/query/method/message/data/dataCashTransaction.json

```diff
@@ -5,62 +5,67 @@
   "type": "object",
   "properties": {
     "object": {
-      "const": "transaction"
+      "const": "coin"
     },
     "action": {
-      "const": "post"
+      "const": "post_transaction"
+    },
+	"transaction_id": {
+        "type": "string",
+        "contentEncoding": "base64",
+        "description": "SHA 256 Hash over base64 of transaction - must be verified"
     },
     "transaction": {
       "type": "object",
-      "description": "the transaction object",
+      "description": "The transaction object",
       "properties": {
         "Version": {
           "type": "integer",
           "description": "The version of the transaction inputs"
         },
-        "TxIn": {
+        "inputs": {
           "type": "array",
           "description": "[Array[Objects]] array of output transactions to use as inputs",
           "items": {
             "type": "object",
             "description": "Object representing a transaction output to use as an input for this transaction",
             "properties": {
-              "TxOutHash": {
+              "tx_out_hash": {
                 "type": "string",
                 "contentEncoding": "base64",
                 "description": "Previous (to-be-used) transaction hash"
               },
-              "TxOutIndex": {
+              "tx_out_index": {
                 "type": "integer",
-                "description": "index of the previous to-be-used transaction"
+                "description": "Index of the previous to-be-used transaction"
               },
-              "Script": {
+              "script": {
                 "type": "object",
                 "description": "The script describing the unlock mechanism",
                 "properties": {
-                  "Type": {
+                  "type": {
                     "type": "string",
                     "description": "Type of script"
                   },
-                  "Pubkey": {
+                  "pubkey": {
                     "type": "string",
                     "contentEncoding": "base64",
                     "description": "The recipient’s public key"
                   },
-                  "Sig": {
+                  "sig": {
                     "type": "string",
                     "contentEncoding": "base64",
                     "description": "Signature on all txins and txouts using the recipient's private key"
                   }
                 },
-                "required": ["Type", "Pubkey", "Sig"]
+                "required": ["type", "pubkey", "sig"]
               }
             },
-            "required": ["TxOutHash", "TxOutIndex", "Script"]
+            "required": ["tx_out_hash", "tx_out_index", "script"]
```

Comment:
Here, if we follow the latest specification ( discussed only on slack ) the only required field is script